### DISCUSSION TITLE
Thanks for creating this awesome package!

### DISCUSSION THREAD

**Discussion by GGAlanSmithee** [isaac-mason/recast-navigation-js#204] [General]
I have been testing any and every emscripten port of recast-detour, trying to find a good work flow (which all includes using UPBGE to generate navmeshes, even with simple geometry) and this package is the only one that "just works" and it works flawlessly so far, in all cases I've tested with react-three-fiber, dynamic generation of meshes obstacles, etc.

**amazing** work, thank you so much for this!!

**Comment by isaac-mason** [comment]
I'm glad to hear it! Let me know if you run into any issues or have suggestions for improvement 😁

**Reply by GGAlanSmithee** [reply]
For sure. I plan on using it in some different scenarios, such as server-side, etc, so if I find anything, I'll report it.

But so far, it's working so nicely! Very performant and accurate, and also more feature-complete than the alternatives, with dynamic obstacles, off-linking, tiling, etc - I am really liking it!
